packages/create-gatsby/src/index.ts

```diff
@@ -181,15 +182,28 @@ ${center(c.blueBright.bold.underline(`Welcome to Gatsby!`))}
 
   enquirer.use(plugin)
 
-  const { name: siteName } = await enquirer.prompt({
-    type: `textinput`,
-    name: `name`,
-    message: `What would you like to call your site?`,
-    initial: `My Gatsby Site`,
-    format: (value: string): string => c.cyan(value),
-  } as any)
+  let data
+  let siteName
+  if (!yesFlag) {
+    ;({ name: siteName } = await enquirer.prompt({
```

Comment:
Shouldn't this have a `const`?

Reply:
Oh wait I see you declared it above. Does that work then? I didn't realise you could assign to an existing variable inside a destructuring like that!